pkgs/development/libraries/boost/generic.nix

```diff
@@ -10,7 +10,7 @@
 , enablePIC ? false
 , enableExceptions ? false
 , enablePython ? hostPlatform == buildPlatform
-, enableNumpy ? false, numpy ? null
+, enableNumpy ? true
```

Comment:
This should default to false. numpy/openblas/gfortran is a bit too much for a default boost installation.

Reply:
numpy is required starting 1.65 because as you can read in the [release notes](http://www.boost.org/users/history/version_1_65_1.html): 

> The boost::python::numeric API has been removed, as it is being obsoleted by boost::python::numpy.

Without numpy extension, it is not possible to manipulate Python array types anymore.

Reply:
Numpy doesn't seem to end up in the closure, so that's good. Its just needed during build-time.

Older versions do not require `numpy`, so can you put this in the function call of `1.65.nix`.